docs/root/start/quick-start/run-envoy.rst

```diff
@@ -79,82 +79,239 @@ The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configu
+         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:76] loading 1 cluster(s)
+         [2020-11-08 12:36:06.546][11][info][config] [source/server/configuration_impl.cc:80] loading 1 listener(s)
+         [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
+         configuration 'my-envoy-config.yaml' OK
+
+   .. tab:: Docker
+
+      .. substitution-code-block:: console
+
+         $ docker run --rm \
+               -v $(pwd)/my-envoy-config.yaml:/my-envoy-config.yaml \
+               envoyproxy/|envoy_docker_image| \
+               --mode validate \
+               -c my-envoy-config.yaml
+         [2020-11-08 12:36:06.543][11][info][main] [source/server/server.cc:583] runtime: layers:
+         - name: base
+           static_layer:
+             {}
+         - name: admin
+           admin_layer:
+             {}
+         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:95] loading tracing configuration
+         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:70] loading 0 static secret(s)
+         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:76] loading 1 cluster(s)
+         [2020-11-08 12:36:06.546][11][info][config] [source/server/configuration_impl.cc:80] loading 1 listener(s)
+         [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
+         configuration 'my-envoy-config.yaml' OK
+
+Envoy logging
+-------------
+
+By default Envoy system logs are sent to ``/dev/stderr``.
+
+This can be overridden using :option:`--log-path`.
+
+.. tabs::
+
+   .. tab:: System
+
+      .. code-block:: console
+
+         $ mkdir logs
+         $ envoy -c envoy-demo.yaml --log-path logs/custom.log
+
+   .. tab:: Docker
+
+      .. substitution-code-block:: console
+
+         $ mkdir logs
+         $ chmod go+rwx logs/
+         $ docker run --rm -it \
+               -p 10000:10000 \
+               -v $(pwd)/logs:/logs \
+               envoyproxy/|envoy_docker_image| \
+               -c /etc/envoy/envoy.yaml \
+               --log-path logs/custom.log
+
+:ref:`Access log <arch_overview_access_logs>` paths can be set for the
+:ref:`admin interface <start_quick_start_admin>`, and for configured
+:ref:`listeners <envoy_v3_api_file_envoy/config/listener/v3/listener.proto>`.
+
+Some Envoy :ref:`filters and extensions <api-v3_config>` may also have additional logging capabilities.
+
+Envoy can be configured to log to :ref:`different formats <config_access_log>`, and to
+:ref:`different outputs <api-v3_config_accesslog>` in addition to files and ``stdout/err``.
+
+.. note::
+
+   If you are running Envoy on a Windows system Envoy will output to ``CON`` by default.
+
+   This can also be used as a logging path when configuring logging.
```

Comment:
@envoyproxy/windows-dev can you confirm the above not about logging with windows is correct ?

Reply:
While stdout will generally refer to CON, that depends on how the envoy process is launched... I'd point out that on windows, logs will default to stdout, and that the Linux /dev/stdout and /dev/stderr filename aliases to FD 1&2 don't exist. The NUL filename on windows can be used in place of /dev/null

Reply:
@wrowe can i land this and then someone with better knowledge could PR with some windows-specific updates ?

there are quite a few places where the docs are posix-focused - im just aware that this config wont work (and not sure what would)

Reply:
We can give you further feedback later this morning. In the posix world, the file `-` name universally represents FD 1 (or FD 0 in the context of an input name). /dev/stdout is actually a newer convention. We need to decide as porters if we will implement the aliases /dev/null, /dev/stdout etc, or whether envoy should have unambiguous, non-os specific implementations of these things.